src/korma_encrypted/core.clj

```diff
@@ -35,15 +35,17 @@
 
 (defn encrypted-field [ent field]
   (-> ent
-      (korma/prepare (fn [v]
-                       (let [unencrypted (get v field)]
-                         (-> v
-                             (assoc (encrypted-name field) (encrypt-value unencrypted))
-                             (dissoc field)))))
-      (korma/transform (fn [v]
+      (korma/prepare (fn [values]
```

Comment:
I think these functions are getting to the point where it might be nice to no longer have them in-line. It would almost make it possible to test the prepare/transform functions directly without having to do the db round-trip.